networkx/algorithms/flow/tests/test_maxflow.py

```diff
@@ -13,13 +13,18 @@
     shortest_augmenting_path,
 )
 
+flow_funcs_without_cutoff = {
+    preflow_push,
+}
+
```

Comment:
can this be imported from `maxflow.py` so that there is a single source of truth?

Reply:
Definitely!